src/Adapter/PaymentAdapter.php

```diff
@@ -18,6 +18,12 @@ public function retrievePayment($paymentId)
         return $this->httpGet($path);
     }
 
+    public function postAuthPayment(array $request)
```

Comment:
$paymentId, $request

Reply:
public function searchWalletTransactions($walletId, array $request)
    {
        $path = "/wallet/v1/wallets/" . $walletId . "/wallet-transactions" . QueryBuilder::build($request);
        return $this->httpGet($path);
    }

Reply:
done